svc-bip-api/src/main/resources/application.properties

```diff
@@ -24,4 +24,5 @@ updateClaimStatusQueue=updateClaimStatusQueue
 getClaimContentionsQueue=getClaimContentionsQueue
 updateClaimContentionQueue=updateClaimContentionQueue
 verifySpecialIssueTypeQueue=confirmCanCallSpecialIssueTypesQueue
+cancelClaimQueue=cancelClaimQueue
```

Comment:
In various places the claim queues are referred to as claim contention queues. Should the naming be consistent?

Reply:
Ah..The naming is fine here! There are two different objects here. `claims` and `contentions`. 👍

Reply:
@agile-josiah claims and contentions are distinctly different things. This PR is aiming to only add cancelClaim, so if you have an example of cancelContention, that would certainly be a problem and would have been an accident on my part